src/_playground/Routes.js

```diff
@@ -285,7 +285,8 @@ export class Routes extends Component {
         super(props);
         this.state = {
             query: '',
-            filteredItems: routes
+            filteredItems: routes,
+            showSideNav: false //for css media queries, default is invisible-'false'
```

Comment:
How is this state variable staying in sync with the CSS media queries?  I only see one place where this value gets set and it's in the `onClick` handler of the toggle button.

Reply:
it adds a class to line 537, frDocsContent depending on state